charts/studio/templates/deployment-studio-backend.yaml

```
@@ -127,13 +132,55 @@ spec:
             - secretRef:
                 name: {{ .Values.studioBackend.envFromSecret }}
             {{- end }}
+        {{- if and .Values.global.basePath (not (eq .Values.global.basePath "/")) }}
+        - name: nginx-sidecar
+          image: "{{ .Values.studioBlobvault.image.repository }}:{{ .Values.studioBlobvault.image.tag }"
+          imagePullPolicy: "IfNotPresent"
+          ports:
+            - name: nginx-http
+              containerPort: 80
+              protocol: TCP
+          livenessProbe:
+            tcpSocket:
+              port: 8080
+            periodSeconds: 10
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 8080
+            periodSeconds: 10
+          volumeMounts:
+            - name: studio-nginx-config
+              mountPath: /etc/nginx/nginx.conf
+              subPath: nginx.conf
+            - name: studio-nginx-sidecar
+              mountPath: /etc/nginx/conf.d/default.conf
+              subPath: default.conf
+          resources:
+            requests:
+              cpu: 10m
+              memory: 64Mi
+            limits:
+              cpu: 100m
+              memory: 256Mi
+          {{- end }}
       volumes:
+        {{ if not .Values.global.blobvault.bucket }}
         - name: blobvault
           persistentVolumeClaim:
             claimName: blobvault
+        {{- end }}
         - name: studio-ca-certificates
           configMap:
             name: studio-ca-certificates
+        {{- if and .Values.global.basePath (not (eq .Values.global.basePath "/")) }}
+        - name: studio-nginx-config
```

```suggestion
        - name: nginx-config
```
We know it relates to studio already based on the file we're in.